tests/test_endpoints.py

```diff
@@ -0,0 +1,223 @@
+from datetime import datetime
+
+import pytest
+
+TEST_PAGE_NAME = f"Test Page - {datetime.now()}"
+
+
+# SETUP
+@pytest.fixture(scope="session")
+def session_page_id(client, page_id) -> str:
+    response = client.pages.create(
+        parent={"page_id": page_id},
+        properties={
+            "title": [{"text": {"content": TEST_PAGE_NAME}}],
+        },
+        # using children = [] for avoiding issue with the Notion API
+        # when creating a subpage without blocks
+        children=[],
+    )
+
+    yield response["id"]
+
+    # at the end of the session, delete the page
+    client.blocks.delete(block_id=response["id"])
+
+
+@pytest.fixture(scope="session")
+def session_block_id(client, session_page_id) -> str:
+    children = [
+        {"paragraph": {"rich_text": [{"text": {"content": "I'm a paragraph."}}]}}
+    ]
+
+    response = client.blocks.children.append(
+        block_id=session_page_id, children=children
+    )
+    assert response["type"] == "block"
+    assert response["results"][0]["type"] == "paragraph"
+
+    return response["results"][0]["id"]
+
+
+@pytest.fixture(scope="session")
+def session_database_id(client, session_page_id) -> str:
+    db_name = f"Test Database - {datetime.now()}"
+    properties = {
+        "Name": {"title": {}},  # required property
+    }
+    title = [{"type": "text", "text": {"content": db_name}}]
+    parent = {"type": "page_id", "page_id": session_page_id}
+    response = client.databases.create(
+        **{"parent": parent, "title": title, "properties": properties}
+    )
+
+    return response["id"]
+
+
+@pytest.fixture(scope="session")
+def session_comment_id(client, session_page_id) -> str:
+    parent = {"page_id": session_page_id}
+    rich_text = [
+        {
+            "text": {
+                "content": "This is a test comment.",
+            },
+        },
+    ]
+
+    response = client.comments.create(parent=parent, rich_text=rich_text)
+
+    assert response["object"] == "comment"
+    assert compare_cleaned_ids(response["parent"]["page_id"], session_page_id)
+    assert response["rich_text"][0]["plain_text"] == rich_text[0]["text"]["content"]
+
+    return response["id"]
+
+
+def compare_cleaned_ids(id1: str, id2: str) -> bool:
+    return id1.replace("-", "").strip() == id2.replace("-", "").strip()
+
+
+# PAGES
```

Comment:
It seems to me that the way you named the tests is quite clear and consistent, so these comments don't feel necessary to me! I understand the appeal but I prefer to make my code files as small as possible. :)